src/thrift_0_14_1/Makefile

```diff
@@ -0,0 +1,31 @@
+SHELL = /bin/bash
+.ONESHELL:
+.SHELLFLAGS += -e -x
+
+THRIFT_VERSION = 0.14.1
+
+MAIN_TARGET = libthrift0_$(THRIFT_VERSION)_$(CONFIGURED_ARCH).deb
+DERIVED_TARGETS = libthrift-dev_$(THRIFT_VERSION)_$(CONFIGURED_ARCH).deb \
+		  python3-thrift_$(THRIFT_VERSION)_$(CONFIGURED_ARCH).deb \
+		  thrift-compiler_$(THRIFT_VERSION)_$(CONFIGURED_ARCH).deb
+
+THRIFT_LINK_PRE = https://archive.apache.org/dist/thrift
```

Comment:
>https://archive.apache.org/dist/thrift [](http://example.com/codeflow?start=18&length=38)

@xumia, will reproducible build help if this download link is down?

Reply:
Yes, wget will work, see line 17, it is only a prefix.